example-react-app/src/app/AppMenu.tsx

```diff
@@ -19,241 +19,241 @@ export const AppMenu = (props: AppMenuProps) => {
+        key={"64303325-6a9a-4013-af3d-72f0a47fc2ed"}
       />
       <MenuItem
         screenId={"IntIdentityIdBrowserCards"}
         icon={<BarsOutlined />}
         caption={"screen.IntIdentityIdBrowserCards"}
-        key={"41f09565-b24c-44da-b44a-6fae439eb6e1"}
+        key={"f5a8102a-d503-4929-8804-ac99157b28f1"}
       />
       <MenuItem
         screenId={"IntIdentityIdBrowserList"}
         icon={<BarsOutlined />}
         caption={"screen.IntIdentityIdBrowserList"}
-        key={"deaa7052-e99d-4638-8680-8e5573abe58b"}
+        key={"4e9e11dd-9896-440a-ada1-c9ad36642829"}
       />
       <MenuItem
         screenId={"StringIdCards"}
         icon={<BarsOutlined />}
         caption={"screen.StringIdCards"}
-        key={"5e7b3304-99d6-4522-bff9-3483b5c8f112"}
+        key={"dc9ec0f1-a1be-4773-be2d-e19cd26a4b7e"}
       />
       <MenuItem
         screenId={"StringIdBrowserCards"}
         icon={<BarsOutlined />}
         caption={"screen.StringIdBrowserCards"}
-        key={"112a82db-d149-44bf-976d-0bb8b7c658ff"}
+        key={"27f8d3fe-895e-4161-af8c-8e21d21895bc"}
       />
       <MenuItem
         screenId={"StringIdBrowserList"}
         icon={<BarsOutlined />}
         caption={"screen.StringIdBrowserList"}
-        key={"0e6e8062-9b0f-4085-8e89-351de5a6e906"}
+        key={"bf16c147-b2a7-41ea-acf3-fecd0379a8ab"}
       />
       <MenuItem
         screenId={"StringIdBrowserTable"}
         icon={<BarsOutlined />}
         caption={"screen.StringIdBrowserTable"}
-        key={"2e1a7ac2-1f22-470f-9ab2-9a980418f3cb"}
+        key={"24426a6a-5f2b-44fc-97a6-2188c48773a8"}
       />
       <MenuItem
         screenId={"WeirdStringIdBrowserCards"}
         icon={<BarsOutlined />}
         caption={"screen.WeirdStringIdBrowserCards"}
-        key={"9651c739-91d0-4571-80a5-35c36ecd0212"}
+        key={"1e023fd1-b6bd-496a-a4b4-dd5b23f89df9"}
       />
       <MenuItem
         screenId={"WeirdStringIdBrowserList"}
         icon={<BarsOutlined />}
         caption={"screen.WeirdStringIdBrowserList"}
-        key={"4a339ccb-d675-46ed-bfe8-7a9c35fd0891"}
+        key={"fc37b77b-07f5-4bbe-8e8a-191902fe9fcf"}
       />
       <MenuItem
         screenId={"WeirdStringIdBrowserTable"}
         icon={<BarsOutlined />}
         caption={"screen.WeirdStringIdBrowserTable"}
-        key={"89980f07-8a0f-42b0-abb4-1a24f26e70db"}
+        key={"2e0f1d16-5cc4-4132-818a-d7b73687d09d"}
       />
       <MenuItem
         screenId={"BoringStringIdBrowserTable"}
         icon={<BarsOutlined />}
         caption={"screen.BoringStringIdBrowserTable"}
-        key={"61e1809f-ac31-4c1e-b66a-23f2c632cd03"}
+        key={"0839a3c5-f3df-4f1d-bdc7-8175075c3c28"}
       />
       <MenuItem
         screenId={"TrickyIdBrowserTable"}
         icon={<BarsOutlined />}
         caption={"screen.TrickyIdBrowserTable"}
-        key={"756c0e44-1aa1-48f7-ac23-a11622a45196"}
+        key={"b1a64e16-dde5-4c8b-9866-2a38009b90d8"}
```

Comment:
`AppMenu` should be excluded from commit if the only changes are re-generated keys.